cmd/checker/checker.go

```diff
@@ -0,0 +1,59 @@
+package checker
+
+import (
+	"kool-dev/kool/cmd/builder"
+	"log"
+)
+
+// Checker defines the check kool dependencies method
+type Checker interface {
+	CheckKoolDependencies() (string, error)
+}
+
+// DefaultChecker holds commands to be checked.
+type DefaultChecker struct {
+	DockerCmd builder.Executor
+	dockerComposeCmd builder.Executor
+}
+
+// NewChecker initializes checker
+func NewChecker() Checker {
+	var (
+		err error
+		dockerInfoCmd *builder.Command
+		dockerComposePsCmd *builder.Command
+	)
+
+	dockerInfoCmd, err = builder.ParseCommand("docker info")
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I think our packages should never "fatal", but instead return an `error`. So the proper exit control is handled in a single layer at the Cobra commands I imagine.